Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -367,6 +416,47 @@ open class Kommunicate: NSObject, Localizable {
             }
         })
     }
+    
+    class func updateSettingsForEmbededMode(viewController: UIViewController) {
```

Comment:
Typo: `Mode`. Place private methods at the end of the file.

Reply:
orgganized Kommunicate file.  Open methods -> Internal Methods -> Private methods -> deprecated methods will be the order